docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -96,6 +96,108 @@ def do_stuff_partitioned():
+day spanning midnight to midnight. For example, defining a partition:
+
+```python file=/concepts/partitions_schedules_sensors/partition_defs.py startafter=start_daily_partition_def endbefore=end_daily_partition_def
+@daily_partitioned_config(start_date="2022-03-12")
+def my_daily_partitioned_config(start: datetime, _end: datetime):
+    return {
+        "ops": {
+            "process_data_for_date": {"config": {"date": start.strftime("%Y-%m-%d")}}
+        }
+    }
+```
+
+will create partitions of the form `(2022-03-12T00:00, 2022-03-13T00:00), (2022-03-13T00:00, 2022-03-14T00:00), ...`
+
+<PyObject object="weekly_partitioned_config" decorator /> - a partition is a week
+spanning Sunday to Sunday. The first partition begins the first Sunday following
+the given start date. For example, defining a partition:
+
+```python file=/concepts/partitions_schedules_sensors/partition_defs.py startafter=start_weekly_partition_def endbefore=end_weekly_partition_def
+@weekly_partitioned_config(start_date="2022-03-12")
+def my_weekly_partitioned_config(start: datetime, _end: datetime):
+    return {
+        "ops": {
+            "process_data_for_date": {"config": {"date": start.strftime("%Y-%m-%d")}}
+        }
+    }
+```
+
+will create partitions of the form `(2022-03-13T00:00, 2022-03-20T00:00), (2022-03-20T00:00, 2022-03-27T00:00), ...`
+
+<PyObject object="monthly_partitioned_config" decorator /> - a partition is a month
+spanning the first to the first. The first partition begins the on the soonest first
+of the month following the given start date. For example, defining a partition:
+
+```python file=/concepts/partitions_schedules_sensors/partition_defs.py startafter=start_monthly_partition_def endbefore=end_monthly_partition_def
+@monthly_partitioned_config(start_date="2022-03-12")
+def my_monthly_partitioned_config(start: datetime, _end: datetime):
+    return {
+        "ops": {
+            "process_data_for_date": {"config": {"date": start.strftime("%Y-%m-%d")}}
+        }
+    }
+```
+
+will create partitions of the form `(2022-04-01T00:00, 2022-05-01T00:00), (2022-05-01T00:00, 2022-06-01T00:00), ...`
+
+You can specify the partitions more granularly using `minute_offset`, `hour_offset`, and `day_offset` in the partitioned config decorator. For example, you can create a hourly partition that splits at the 15 minute mark by doing the following:
+
+```python file=/concepts/partitions_schedules_sensors/partition_defs.py startafter=start_hourly_offset_partition_def endbefore=end_hourly_offset_partition_def
+@hourly_partitioned_config(start_date=datetime(2022, 3, 12), minute_offset=15)
+def my_hourly_offset_partitioned_config(start: datetime, _end: datetime):
+    return {
+        "ops": {
+            "process_data_for_date": {"config": {"date": start.strftime("%Y-%m-%d")}}
+        }
+    }
+```
+
+You can have a weekly partiton that splits on each Wednesday at 4:15pm by writing the following:
```

Comment:
Typo: `partiton`. Also, should probably PartitionsDefinition for the same reason mentioned above.